src/main/java/com/google/codeu/servlets/AboutMeServlet.java

```diff
@@ -43,7 +43,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
             return;
         }
 
-        User userData = datastore.getUser(user);
+        User userData = datastore.getUser("test@example.com");
```

Comment:
Is this intentional?

Reply:
I changed it for debugging, and forgot to change it back.